beater/beater.go

```diff
@@ -44,16 +43,19 @@ func (bt *beater) Run(b *beat.Beat) error {
 	}
 
 	server := newServer(bt.config, callback)
-	start(server, bt.config.SSL)
+	bt.errCh = start(server, bt.config.SSL)
 	defer stop(server)
 
 	logp.Info("apm-server is running! Hit CTRL-C to stop it.")
```

Comment:
Hm,  I see. It's still the case that the log is potentially printed before the http server is running, but don't know how we could do it otherwise as it happens in a go routine and then is blocking. 

Perhaps we should rephrase the log to "started" as I would say running means the http listener is able to receive calls?